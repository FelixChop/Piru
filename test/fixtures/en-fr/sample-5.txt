Despite his gregarious nature, he often felt like a fish out of water at formal soirÃ©es.
