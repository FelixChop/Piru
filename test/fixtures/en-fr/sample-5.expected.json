[
  {
    "word": "gregarious",
    "lemma": "gregarious",
    "definition": "très sociable",
    "citation": "Despite his gregarious nature, he often felt like a fish out of water at formal soirées.",
    "to_learn": true
  },
  {
    "word": "fish out of water",
    "lemma": "fish out of water",
    "definition": "se sentir mal à l'aise",
    "citation": "Despite his gregarious nature, he often felt like a fish out of water at formal soirées.",
    "to_learn": true
  }
]
